src/components/sbb-link-list/sbb-link-list.tsx

```diff
@@ -35,53 +53,76 @@ export class SbbLinkList {
    */
   @Prop() public titleLevel?: InterfaceTitleAttributes['level'] = '2';
 
+  private _getClassString(): string {
+    let horizontalClass = this.horizontalFrom
+      ? ` link-list--horizontal-from-${this.horizontalFrom}`
+      : '';
+
+    if (!horizontalClass) {
+      horizontalClass = this.orientation === 'horizontal' ? ' link-list--horizontal' : '';
+    }
+
+    return `link-list${horizontalClass}`;
+  }
+
   /**
-   * Choose the link list style. This
-   * does not refer to light or dark
-   * mode, but the background color
-   * on which the list is placed. Light
-   * and dark mode styling will be applied
-   * differently.
+   * Create an array with only the sbb-link-children
+   * @private
    */
-  @Prop() public variant: InterfaceLinkListAttributes['variant'] = 'positive';
+  private _readLinks(): void {
+    this._links = Array.from(this._element.children).filter(
+      (e): e is HTMLSbbLinkElement => e.tagName === 'SBB-LINK'
+    );
+  }
 
   public componentWillLoad(): void {
```

Comment:
Should this maybe be done in the `connectedCallback` instead of `componentWillLoad` lifecycle?
What do you think?

Reply:
I agree. :)